modules/olmv1-adding-a-catalog.adoc

```diff
@@ -35,9 +35,11 @@ spec:
     image:
       ref: registry.redhat.io/redhat/redhat-operator-index:v{product-version} <1>
       pullSecret: <pull_secret_name> <2>
+      pollInterval: 30m <3>
 ----
 <1> Specify the catalog's image in the `spec.source.image` field.
 <2> If your catalog is hosted on a secure registry, such as `registry.redhat.io`, you must create a pull secret scoped to the `openshift-catalog` namespace.
+<3> Specify the interval for polling the remote registry for newer image digests. The default value is `24h`. Valid units include `s`, `m`, and `h`. To disable polling, set a zero value, such as `0s`.
```

Comment:
Though it is quite easy, do you think it could be a good idea to specify seconds, minutes, hours? Up to you :)